src/orquestra/sdk/schema/ir.py

```diff
@@ -160,6 +160,7 @@ class Resources(BaseModel):
     memory: t.Optional[str] = None
     disk: t.Optional[str] = None
     gpu: t.Optional[str] = None
+    nodes: t.Optional[int] = None
```

Comment:
**suggestion**: I know we haven't done that in the past, but could you document what's the meaning behind this field, and what's the effect of having here values like `None`, `-1`, `0`, `1`, `2`, etc?